src/nvim/fileio.c

```diff
@@ -4333,108 +4333,110 @@ void shorten_fnames(int force)
   redraw_tabline = TRUE;
 }
 
-/*
- * add extension to file name - change path/fo.o.h to path/fo.o.h.ext
- *
- * Assumed that fname is a valid name found in the filesystem we assure that
- * the return value is a different name and ends in 'ext'.
- * "ext" MUST be at most 4 characters long if it starts with a dot, 3
- * characters otherwise.
- * Space for the returned name is allocated, must be freed later.
- * Returns NULL when out of memory.
- */
-char_u *
-modname (
-    char_u *fname,
-    char_u *ext,
-    int prepend_dot                /* may prepend a '.' to file name */
-)
+/// Get new filename ended by given extension.
```

Comment:
There's not 100% consensus on that. At the very beginning, the advice on descriptive voice for function docs made into the style guide. 
But we have also been advocating imperative voice almost for everything else (commit messages, PR titles, etc). And many people (me included have also been using imperative voice for function docs).
Personally, I think that's something we should change in the style guide
I much prefer imperative voice in general. It leads to shorter and clearer phrases most of the time.
But that's my opinion. I know there are people that think just the opposite.